src/commcare_cloud/ansible/letsencrypt_cert.yml

```diff
@@ -41,11 +41,11 @@
 
     - name: Run Certbot command when there is a www host
       command: "certbot certonly --webroot -w /var/www/letsencrypt/ -m {{ root_email }} --agree-tos -n -d {{SITE_HOST}} -d {{ NO_WWW_SITE_HOST }}{% if ALTERNATE_HOSTS %} -d {{ ALTERNATE_HOSTS | join(' -d ') }}{% endif %} --expand"
-      when: NO_WWW_SITE_HOST|bool
+      when: NO_WWW_SITE_HOST is defined and NO_WWW_SITE_HOST
 
     - name: Run Certbot command when there is no www host
       command: "certbot certonly --webroot -w /var/www/letsencrypt/ -m {{ root_email }} --agree-tos -n -d {{SITE_HOST}}{% if ALTERNATE_HOSTS %} -d {{ ALTERNATE_HOSTS | join(' -d ') }}{% endif %} --expand"
-      when: not NO_WWW_SITE_HOST
+      when: NO_WWW_SITE_HOST is not defined or not NO_WWW_SITE_HOST
```

Comment:
I don't believe there was an issue with this conditional, but I thought it prudent to keep both expressions logically orthogonal